src/bench/ss_bench/operations/get.rs

```diff
@@ -100,7 +101,17 @@ impl Operations {
                 let mut sizes: Vec<usize> = vec![];
                 for key in keys {
                     let start = Instant::now();
-                    let val_size = match store.get(&key, u64::MAX).await.unwrap() {
+                    let val_size = match store
+                        .get(
+                            &key,
+                            ReadOptions {
```

Comment:
how about use the same `ReadOptions` with `Hummock`? (for prefetch)

Reply:
You mean reusing the existed hummock ReadOptions for state store interfaces?

Reply:
No. In the future we may need executor to tell us which it is a large scan request so that storage can decide whether we need to prefetch more data from object storage.